### PR TITLE
Format all the dates similar to CLI datetime representation

### LINKED ISSUE #165: Difference in datetime formatting between UI and CLI
**UI:** Jul 1, 2015 11:12:26 PM
**CLI:** Wed, 01 Jul 2015 23:12:26 UTC

### PR DESCRIPTION
**Pull request by enykeev**
Resolves #165

### CODE REVIEW

**Comment by Kami** [comment]
LGTM

File: apps/st2-actions/template.html [hunk 1]
```diff
@@ -184,7 +184,7 @@ <h2 class="st2-details__panel-title">
                   <div class="st2-label" status="record.status"></div>
                 </div> -->
                 <div class="st2-flex-table__column st2-actions__details-column-time">
-                  {{ record.start_timestamp | date:'medium':'UTC' }}
+                  {{ record.start_timestamp | date:'EEE, dd MMM y HH:mm:ss UTC':'UTC' }}
```
Comment (Kami) [inline apps/st2-actions/template.html]: Minor thing - can we make this a constant and re-use it through the code base?

Or that is not possible, I guess, another option is to create a custom filter function...

Comment (enykeev) [inline apps/st2-actions/template.html]: I was thinking about it but decided to postpone until we find a way to switch between TZ

**Comment by lakshmi-kannan** [comment]
Wondering why this isn't merged? @enykeev @Kami

**Comment by Kami** [comment]
No idea.

LGTM, +1

**Comment by enykeev** [comment]
> No idea.

Me neither. Merging.

**Pull request merged**